modules/window_manager/index.html

```diff
@@ -0,0 +1,16 @@
+<head>
+	<link rel="stylesheet" href="res\css\main.css">
```

Comment:
You're using \ instead of / here. I'm not really sure why that works (Chrome seems to convert \ into /) but I'm pretty sure it will only work in the demo, not in an installed version of Airborn OS